app/views/components/gds/gdsMainTemplate.scala.html

```diff
@@ -52,6 +52,15 @@
     <meta name="format-detection" content="telephone=no" />
     <!--[if lte IE 8]><link href='@controllers.routes.Assets.versioned("stylesheets/application-ie-8.css")' rel="stylesheet" type="text/css" /><![endif]-->
     <!--[if gt IE 8]><!--><link href='@routes.Assets.versioned("stylesheets/customsdecexfrontend-app.css")' media="all" rel="stylesheet" type="text/css" /><!--<![endif]-->
+
+    <noscript>
+      <style>
+        .govuk-form-group--error {
+          padding-left: 0;
+          border-left: none;
+        }
+      </style>
+    </noscript>
```

Comment:
Should this be here in the main gdsTemplate or could it just be on the file upload page itself (that is the only page that needs it)?

Reply:
Since it's styling it has to be placed within the `<head>` tag, which is in the main template only. It won't affect the other pages btw